### DISCUSSION TITLE
Idle settings and gaps in track

### DISCUSSION THREAD

**Discussion by bennettscience** [OpenTracksApp/OpenTracks#1870] [Q&A] [answered]
I think this is definitely user error, but what settings work well to avoid gaps in tracks? Particularly, in hiking or walking.

I used OpenTracks and a second tracking app on a hike and got two vastly different distances and hike paths on the map. I feel like it has to do with the idle setting - my guess is that I stopped for the timeout but started walking again before another timeout check was performed, resulting in gaps. Here are two screenshots showing the difference in recorded activity:

OpenTracks in OSM Dashboard
![Screenshot_20240225-171719](https://github.com/OpenTracksApp/OpenTracks/assets/3878010/6bc6d0cf-88db-4060-b8cb-7b89c5544bb3)

EBird
![Screenshot_20240225-171744](https://github.com/OpenTracksApp/OpenTracks/assets/3878010/491495aa-0deb-42aa-a7bc-cdaf25b38648)

Has anyone else seen similar behavior?

OpenTracks v4.11.2 5683
OnePlus N100

**Comment by dennisguse** [comment, marked as answer]
Please check your configuration: Settings -> GPS -> `GPS Accuracy`, `Recording distance interval`, and `Idle threshold`.
Likely those need to be increased for your device / activity (looks like hiking through the woods).
And your GPS might not be as precise as you believe it is...

**Comment by dennisguse** [comment]
OSMDashboard would show a symbol at every location you became idle.
(not sure if this has to be configured though)

**Comment by bennettscience** [comment]
Thanks @dennisguse. I messed around more yesterday where I know I have solid connection and precise GPS locations and I think I've got my idle timer set more correctly. I also lowered the distance interval and that seemed to help. I figured it was user error - thanks for the tips and for making such a great tracker.
